docs/content/contributing/bicep/bicep-contribution-flow/owner-contribution-flow.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Owner Contribution Flow
+geekdocNav: true
+geekdocAlign: left
+geekdocAnchor: true
+---
+
+This section describes the contribution flow for module owners who are responsible for creating and maintaining Bicep Modules.
+
+{{< toc >}}
+
+<br>
+
+{{< hint type=important >}}
+
+This contribution flow is for **Module Owners** only.
+
+As a **Bicep Module Owner** you need to be aware of the [AVM Contribution Process Overview](https://azure.github.io/Azure-Verified-Modules/contributing/process/), [Shared Specifications](https://azure.github.io/Azure-Verified-Modules/specs/shared/) (including [Interfaces](https://azure.github.io/Azure-Verified-Modules/specs/shared/interfaces/)) and [Bicep-specific](https://azure.github.io/Azure-Verified-Modules/specs/bicep/) specifications as as these need to be considered during pull request reviews for the modules you own. The purpose of this **Owner Contribution Flow** is to simplify and list all activities as an owner and to help you understand your responsibilities as an owner.
+
+{{< /hint >}}
+
+<br>
+
+---
+
+<br>
+
+### 1. Owner Activities and Responsibilities
+
+<!-- TODO: Rephrase sections if required -->
+Familiarise yourself with the responsibilities as **Module Owner** outlined in [Team Definitions & RACI](https://azure.github.io/Azure-Verified-Modules/specs/shared/team-definitions/#module-owners) and [Module Owner Responsibilities](https://azure.github.io/Azure-Verified-Modules/help-support/issue-triage/brm-issue-triage/#module-owner-responsibilities) in the [BRM Issue Triage](https://azure.github.io/Azure-Verified-Modules/help-support/issue-triage/brm-issue-triage/).
+
+1. Create GitHub teams as outlined in [SNFR20](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only) and add respective parent teams:
```

Comment:
We will want to modify SNFR20 to include the following:

Link the contributors Microsoft Alias to their GitHub account. - https://repos.opensource.microsoft.com/link
Once linked they will need to join the "Azure" organization - https://repos.opensource.microsoft.com/orgs/Azure

Reply:
Done